crates/astria-conductor/src/sequencer/block_stream.rs

```diff
@@ -115,7 +115,7 @@ impl BlocksFromHeightStream {
     )]
     pub(super) fn set_latest_observed_height_if_greater(&mut self, height: Height) {
         if !self.heights.set_latest_observed_if_greater(height) {
-            info!("observed latest sequencer height older than previous; ignoring it");
+            info!("observed latest sequencer height older or the same as previous; ignoring it");
```

Comment:
I am wondering if this should be `debug!` instead of `info!`. On the other hand, it is arguably one of the core purposes of conductor to observe (and hence) the current height of sequencer.

Reply:
i think info is fine, if we see it polluting we can change it but i don't see it too often